include/NovelRT/Audio/AudioService.h

```diff
@@ -30,7 +29,7 @@ namespace NovelRT {
     SoundBank _sounds;
     MusicBank _music;
     ChannelMap _channelMap;
-    NovelLoggingService _logger;
+    LoggingService _logger;
```

Comment:
I'd like to better understand why `LoggingService` is being used as a value type

Reply:
@capnkenny ?

Reply:
Hmm...after reading through master, it appears I used it as a value instead of reference type throughout the code base. I believe that I had switched it due to having issues with only using one pointer....however specifics I do not remember at this time 😅. 

Also, I believe we had planned to switch it to reference after the NovelRunner restructure from #87

(That's IF I understood this question correctly >.>)

Reply:
Propagated to #101 . Resolving.